core/markets/network/CMakeLists.txt

```diff
@@ -3,10 +3,10 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-add_library(libp2p_storage_market_network
-    libp2p_storage_market_network.cpp
+add_library(libp2p_market_network
```

Comment:
Not sure that was the right choice to have targets prefixed `libp2p_` being created in fuhon project.

Reply:
fixed